compiler/record-minmax-conversion-test/gen_h5_random_inputs.py

```diff
@@ -22,7 +22,6 @@
 
 # Build TFLite interpreter. (to get the information of model input)
 interpreter = tf.lite.Interpreter(model)
-interpreter.allocate_tensors()
```

Comment:
This line is removed because it it for interpreting, isn't it?

Reply:
The interpreter is used only for getting the input information, so tensors need not be allocated.